extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-attachment.js

```diff
@@ -222,5 +225,55 @@ describes.realWin(
       const {activeProductData} = storeService.get(StateProperty.SHOPPING_DATA);
       expect(activeProductData).to.be.null;
     });
+
+    async function setupPDP() {
+      await layoutShoppingImplAndAttachmentChildImpl();
+      // Override to simulate data for one product on the page.
+      storeService.dispatch(Action.ADD_SHOPPING_DATA, {
+        page1: {
+          [shoppingData.productId]: shoppingData.items[0],
+        },
+      });
+      storeService.dispatch(Action.TOGGLE_PAGE_ATTACHMENT_STATE, true);
+    }
+
+    it('should call analytics service on PLP card click', async () => {
```

Comment:
nit: please be more specific with these test names. They should reflect the assertion.
If it was truly "should call analytics service" then the assertion would be "expect(trigger).to.have.been.called;"

This test could be smth like "should call PDP view analytics event on PLP card click"